src/actions/AccountReferralActions.ts

```diff
@@ -110,10 +110,14 @@ export function activatePromotion(installerId: string): ThunkAction<Promise<void
       return
     }
     if (!reply.ok) {
-      console.warn(`Referral server returned status code ${reply.status}`)
-    }
-    if (reply.status === 404) {
-      throw new Error(`Invalid promotion code ${installerId}`)
+      const text = await reply.text()
+      console.warn(text)
```

Comment:
Shouldn't we prefix the warn with something so the dev knows where the log is coming from?